### PR TITLE
ParzivalAFK Entry 1

### PR DESCRIPTION
**Pull request by ParzivalAFK**
For this submission I created half of the ball from straight lines optical illusion.

### CODE REVIEW

File: Art/ParzivalAFK/HalfSpinPoints.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+.ball_6{
+    animation-delay: 5s;
+}
+.ball_7{
+    animation-delay: 6s;
+}
+.ball_8{
+    animation-delay: 7s;
+} */
```
Comment (sophiabrandt) [inline Art/ParzivalAFK/HalfSpinPoints.css:83]: Please remove out-commented code.

File: include.js [hunk 2]
```diff
@@ -2037,6 +2037,13 @@ let cards = [
     imageLink: './Art/SamilAbud/bouncing-effect.gif',
     author: 'Samil Abud',
     githubLink: 'https://github.com/samilabud'
+  },
+  {
+    artName: 'Half Spin Points',
+    pageLink: './Art/ParzivalAFK/HalfSpinPoints.html',
+    imageLink: './Art/ParzivalAFK/HalfSpinPoints.gif',
```
Comment (sophiabrandt) [inline include.js:2044]: That gif does not exist in your contribution. The file you commited is called `Animation1.gif`. You can either rename the file, or the link to the file.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: include.js (hunk 2 above)
Comment (ParzivalAFK) [reply include.js]: Great catch, thank you so much!

File: Art/ParzivalAFK/HalfSpinPoints.css (hunk 1 above)
Comment (ParzivalAFK) [reply Art/ParzivalAFK/HalfSpinPoints.css]: Done thank you.

**Comment by ParzivalAFK** [comment]
I believe I fixed my Merge Conflict. I am sorry for the trouble I am brand new to this.

Thank you for all that you do!

**Pull request merged**